app/eth-manage.hoon

```diff
@@ -34,7 +34,7 @@
       %look-ethnode
     :_  ~
     =/  pul
-      (need (de-purl:html 'http://eth-mainnet.urbit.org:8545'))
+      (need (de-purl:html 'http://localhost:8545'))
```

Comment:
This looks like an artifact left in from local testing. I suspect it might break some production ships if we merge it.